src/Filters/PaginatorPageFieldValidator.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Filters;
+
+class PaginatorPageFieldValidator
```

Comment:
Maybe rename class to `PaginatorValidator` and the method to `isPageNumberValid`?